10/umbraco-cms/fundamentals/setup/install/unattended-install.md

```diff
@@ -48,6 +48,10 @@ It is recommended that you make use of the values shown below for the `Cache`, `
 }
 ```
 
+{% hint style="info" %}
+The 'umbracoDbDSN_ProviderName' attribute sets the .NET Framework data provider name for the DataSource control's connection. For more information on the data providers included in the .Net Framework, see the [Microsoft Documentation](https://learn.microsoft.com/en-us/dotnet/api/system.web.ui.webcontrols.sqldatasource.providername?view=netframework-4.8.1#remarks).
```

Comment:
removed the versions so that we do not need to change this in the future. The way I have suggested will make sure it will always be on latest version of the Microsoft article (not just 4.8.1)
```suggestion
The 'umbracoDbDSN_ProviderName' attribute sets the .NET Framework data provider name for the DataSource control's connection. For more information on the data providers included in the .Net Framework, see the [Microsoft Documentation](https://learn.microsoft.com/en-us/dotnet/api/system.web.ui.webcontrols.sqldatasource.providername?#remarks).
```

Reply:
I think  this should be added here: https://docs.umbraco.com/umbraco-cms/fundamentals/setup/install/unattended-install#sql-server-example-in-appsettings.json as it is related to SQL example